.github/workflows/specs.yml

```diff
@@ -0,0 +1,81 @@
+name: Test extractions for `hacpsec/specs`
+
+on:
+  push:
+    branches:
+    - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: cachix/install-nix-action@v20
+    - uses: cachix/cachix-action@v12
+      with:
+        name: hacspec
+        authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+    - name: Build
+      run: nix build
+
+    - name: Install
+      run: |
+        nix profile install nixpkgs#yq
```

Comment:
Please document in #4 that this is using nix for now and that we need to add this without nix as well.

Reply:
(I edited #4)

Reply:
We'll need to make sure that things work for everyone. But let's discuss and do something in a follow up.